containers/init-image/Dockerfile

```diff
@@ -16,6 +16,21 @@ LABEL org.opencontainers.image.vendor="Uyuni project"
 LABEL org.opencontainers.image.url="https://www.uyuni-project.org/"
 # endlabelprefix
 
+# Create stable static UID and GID for salt, tomcat, apache (wwwrun), postgres, ...
+RUN groupadd -r --gid 10550 susemanager && \
+  groupadd -r --gid 10551 tomcat && \
+  groupadd -r --gid 10552 www && \
+  groupadd -r --gid 10553 wwwrun && \
+  groupadd -r --gid 10554 salt && \
+  groupadd -r --gid 10555 tftp && \
+  groupadd -r --gid 10556 postgres
+
+RUN useradd -r -s /usr/sbin/nologin -G susemanager,www -g tomcat -d /usr/share/tomcat --uid 10551 tomcat && \
+  useradd -r -s /usr/sbin/nologin -G susemanager,www -g wwwrun -d /var/lib/wwwrun --uid 10552 wwwrun && \
+  useradd -r -s /usr/sbin/nologin -G susemanager -g salt -d /var/lib/salt --uid 10554 salt && \
+  useradd -r -s /usr/sbin/nologin -g tftp -d /srv/tftpboot --uid 10555 tftp && \
+  useradd -r -s /usr/bin/bash -g postgres -d /var/lib/pgsql --uid 10556 postgres
+
```

Comment:
I think we will need to add yet another image: this init-image will not be needed for SUSE Manager since we would be using BCI:Init image for it.

Reply:
My idea was to have this init image for both uyuni (based on Leap or whatever there is) and also SUMA based on BCI:Init for start and then something more lightweight.

That's why this is using shadow utils and not a systemd-sysusers.

Once systemd is not needed, we just drop it from this image and use it only for some really default config across all the images.

Reply:
To use the base image for both Uyuni and SUMA, we need a special (new) base image that would use either our `init-image` for uyuni or `BCI:Init` for SUSE Manager. It would only be a parameter to change to base image. The current `init-image` in our repo is only replacing the missing Leap init image that would be the equivalent of BCI:Init.

Reply:
At [my branch project](https://build.opensuse.org/package/show/home:oholecek:branches:systemsmanagement:Uyuni:Master:ServerContainer/init-image) I switched init-image source from `opensuse/leap:15.5` to `bci/bci-init:15.5` to test if it builds and indeed all our actions are basically no-ops (see [build log details](https://build.opensuse.org/package/live_build_log/home:oholecek:branches:systemsmanagement:Uyuni:Master:ServerContainer/init-image/containers/x86_64)).

So I think we can go ahead with the plan to use our `init-image` for uyuni (based on `opensuse/leap`) and also for suma (based on `bci/bci-init`).